test/jasmine/tests/command_test.js

```diff
@@ -594,7 +594,8 @@ describe('attaching component bindings', function() {
             expect(gd.layout.sliders[0].active).toBe(0);
 
             // Check that it still has one attached listener:
-            expect(typeof gd._internalEv._events.plotly_animatingframe).toBe('function');
+            expect(typeof gd._internalEv._events.plotly_animatingframe).toBe('function',
+                gd._internalEv._events.plotly_animatingframe);
```

Comment:
The second argument to `toBe` adds a message? No worries, just checking to make sure there's no unintended difference in function.